src/tracks/playabletrack/wavetrack/ui/WaveTrackShifter.cpp

```diff
@@ -52,8 +66,12 @@ class WaveTrackShifter final : public TrackShifter {
          auto data =
             static_cast<WaveTrack::IntervalData*>( myInterval.Extra() );
          auto clip = data->GetClip().get();
-         return !(clip->IsClipStartAfterClip(interval.Start()) ||
```

Comment:
This member function of WaveClip becomes unused.  Remove it?

Reply:
It's removed in smart clips PR

Reply:
ok